jenkins/release.jenkinsFile

```diff
@@ -0,0 +1,16 @@
+lib = library(identifier: 'jenkins@1.5.3', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+standardReleasePipelineWithGenericTrigger(
+    tokenIdCredential: 'jenkins-performance-analyzer-commons-generic-webhook-token',
+    causeString: 'A tag was cut on opensearch-project/performance-analyzer-commons repository causing this workflow to run',
+    downloadReleaseAsset: true,
+    publishRelease: true) {
+        publishToMaven(
+            signingArtifactsPath: "$WORKSPACE/repository/",
+            mavenArtifactsPath: "$WORKSPACE/repository/",
+            autoPublish: true
```

Comment:
Needs to be `false` just for the first release just to make sure it passes all checks and has all artifacts as intended. Similar for line 10 too